packages/grid_client/src/modules/models.ts

```diff
@@ -142,6 +142,7 @@ class MachineModel {
   @Expose() @IsInt() @IsOptional() solutionProviderId?: number;
   @Expose() @IsString() @IsOptional() zlogsOutput?: string;
   @Expose() @IsString({ each: true }) @IsOptional() gpus?: string[];
+  @Expose() @IsString({ each: true }) @IsOptional() features?: string[];
```

Comment:
do we need this ?

Reply:
yes we do as it might be used like [this](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/23d742a24802a246c795b2e13fea029d214c9015/packages/grid_client/scripts/single_vm_zos4.ts#L89)